src/server/models/users.model.js

```diff
@@ -81,6 +81,7 @@ let userSchema = new Schema({
   job_position: String,
   about: String,
   languages: String,
+  requestGet: Boolean,
```

Comment:
I think the name of this value need to be more accurate.
Now is little bit ambiguous to me.